lang/en.json

```diff
@@ -57,6 +57,8 @@
     "CMPBrowser.size": "Size",
     "CMPBrowser.spellBrowser":"Spell Browser",
     "CMPBrowser.npcBrowser":"NPC Browser",
+    "CMPBrowser.featBrowser": "Feat Browser",
+    "CMPBrowser.itemBrowser": "Item Browser",
     "CMPBrowser.settings":"Settings"
-    
-}
\ No newline at end of file
+
+}

```

Comment:
You've added references to CMPBrowser.rogue, .fighter, .monk without defining them in any of the provided language files.